src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java

```diff
@@ -234,6 +235,15 @@ protected void submit(StaplerRequest req, StaplerResponse rsp) throws IOExceptio
         }
     }
 
+    @Override
+    protected @Nonnull String getSuccessfulDestination() {
+        FolderComputation<I> computation = getComputation();
+        if (computation != null) {
```

Comment:
@jglick Before to do some tests and dive into the source code...

```
@Override
protected @Nonnull String getSuccessfulDestination() {
    FolderComputation<I> computation = getComputation();
    if (computation != null) {
        return ((FolderComputation) computation).getSearchUrl() + "console";
    }
    return super.getSuccessfulDestination();
}
```
This seems it works fine. `computation` is never `null` when this method is invoked because:

1. Adding a new `ComputedFolder`, a  `computation` is created here:

`init()` call to `loadComputation()` and this one is:

```
private synchronized void loadComputation() {
    try {
        FileUtils.forceMkdir(getComputationDir());
    } catch (IOException x) {
        LOGGER.log(Level.WARNING, null, x);
    }
    computation = createComputation(null);
    XmlFile file = computation.getDataFile();
    if (file != null && file.exists()) {
        try {
            file.unmarshal(computation);
        } catch (IOException e) {
            LOGGER.log(Level.WARNING, "Failed to load " + file, e);
        }
    }
}
```

1. Editing a `ComputedFolder`, `computation` always exists from the XML file.